src/python/py_whisper_pipeline.cpp

```diff
@@ -5,7 +5,7 @@
 #include <pybind11/stl.h>
 #include <pybind11/stl_bind.h>
 
-#include "tokenizers_path.hpp"
+#include "tokenizer/tokenizer_path.hpp"
 #include "openvino/genai/whisper_generation_config.hpp"
 #include "openvino/genai/whisper_pipeline.hpp"
```

Comment:
Why do you modify only src folder and not include?

Reply:
what are recommendations for `include`?

Regarding Whisper specifically, I would move WhisperGenerationConfig inside the pipeline class, as it's done for text 2 image:
https://github.com/openvinotoolkit/openvino.genai/blob/ad2faf53c5a5e9c17a5eae9c1242c6ef6040a87b/src/cpp/include/openvino/genai/text2image/pipeline.hpp#L50-L70

CC @as-suvorov

Reply:
Yes and

1. continuous_batching
  cache_eviction.hpp, continuous_batching_pipeline.hpp (rename to pipeline.hpp), continuous_batching_pipeline.hpp scheduler_config.hpp
4. llm
  llm_pipeline.hpp (rename to pipeline.hpp)
6. lora
  lora_adapter.hpp BTW, should the files be renamed here and in `src`?
8. tokenizer
  tokenizer.hpp

Reply:
regarding 2-3 - why do we need to create a folder with a single file?

Reply:
Consistency. The number of symbols would stay exactly the same in both versions of include.